### PR TITLE
Feature branch 1

### PR DESCRIPTION
**Pull request by nikoescobal**
+Given space between sidebar and embedded video.

### CODE REVIEW

**Comment by elmejdki** [comment]
## Pull Request under review by _Zakariae El Mejdki_

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+ 
+ 
+
+## Authors 
+____________________________________________________
+#### Muhammed Cihad Kurel
+Github: [@mcihadkurel](https://github.com/mcihadkurel)
+____________________________________________________
+#### Nikolas Escobal
+Github: [@nikoescobal](https://github.com/nikoescobal/Youtubeclone-muhammed-niko/commits?author=nikoescobal)
```
Comment (elmejdki) [inline README.md]: **[OPTIONA]** add your LinkedIn and Twitter profile links

File: index.html [hunk 2]
```diff
@@ -0,0 +1,660 @@
...
+  <body>
+    <nav class="flex">
+      <div id="left-navbar" class="flex">
+        <i class="fa fa-bars" aria-hidden="true"></i>
+        <img
+          src="assets/YouTube-Logo.svg"
+          style="width: 80px;"
+          height:80px;
+          alt=""
+        />
```
Comment (elmejdki) [inline index.html]: Please surround your images with a link tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,660 @@
...
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Youtube Page</title>
+  </head>
+  <body>
+    <nav class="flex">
+      <div id="left-navbar" class="flex">
+        <i class="fa fa-bars" aria-hidden="true"></i>
```
Comment (elmejdki) [inline index.html]: please change this into an imbeded image, like the one below it.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,660 @@
...
+        <button id="search-button">
+          <i class="fa fa-search" aria-hidden="true"></i>
+        </button>
+      </div>
+
+      <div id="right-navbar" class="flex">
+        <i class="fa fa-video-camera" aria-hidden="true"></i>
+        <i class="fa fa-th fa-lg" aria-hidden="true"></i>
+        <i class="fa fa-bell" aria-hidden="true"></i>
+        <i class="fa fa-envelope" aria-hidden="true"></i>
```
Comment (elmejdki) [inline index.html]: these icons also change them into embedded images.
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,660 @@
...
+              </p>
+              <p class="aside-views">
+                366M Views • 5 Years Ago
+              </p>
+            </div>
+          </div>
+        </aside>
+      </div>
+    </div>
+    <div class="footer"></div>
```
Comment (elmejdki) [inline index.html]: please use semantic HTML, change this line into footer tag and also add the main tag to your code.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+body {
+  background-color: rgba(200, 200, 200, 0.1);
+  font-family: Roboto, Arial, “Sans Serif”, Noto, “YT Sans”, “Roboto Mono”;
+}
+
+/* .video { */
+/* margin-left: initial; */
+/* padding-left: 100px; */
+/* padding-top: 30px; */
+/* } */
```
Comment (elmejdki) [inline style.css]: Please remove all the comments from your CSS files and HTML.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,660 @@
...
+      <div class="main-fc d-flex flex-column">
+        <div class="container">
+          <div class="main-video">
+            <iframe
+              width="853"
+              height="480"
+              src="https://www.youtube.com/embed/YbYWhdLO43Q"
+              frameborder="0"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
```
Comment (elmejdki) [inline index.html]: Please remove all inline styling and use your CSS instead.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+  align-self: center;
+}
+
+#autoplay {
+  display: inline;
+}
+
+.inline {
+  display: inline;
+}
```
Comment (elmejdki) [inline style.css]: this is duplicate you can use only the class if you want please remove the id selector

File: style.css [hunk 9]
```diff
@@ -0,0 +1,235 @@
...
+  position: relative;
+}
+.aside-video {
+  grid-template-columns: 180px 180px;
+  display: grid;
+  margin-bottom: 4px;
+}
+.aside-description {
+  font-family: Roboto;
+  line-height: 1.2em;
```
Comment (elmejdki) [inline style.css]: you are duplicating these line of codes too much of times try to give it it's own selector and uses it on your HTML instead

File: style.css [hunk 10]
```diff
@@ -0,0 +1,235 @@
...
+.icon-spacing {
+  padding: 30px 15px;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+.user-comments {
+  display: grid;
+}
```
Comment (elmejdki) [inline style.css]: this one also is used below in grid-container selector.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Greate job so far guys :clap:, but your project needs some small changes in order to get approved :tophat: 
### status: require changes

- Please add live demo link to your repo you can use [githack](https://raw.githack.com/)
- Please add a .gitignore file into the root of your repo and add ".DS_Store" inside it, so that file will be ignored by Github.
image

image

- Please align these two divs in front of each other you can do it easily with flexbox, I mean (**up next** and **autoplay**):
image

- Please remove CSS duplicated code.

### please fix all the required changes in here and below, before requesting another code review.
if you have any questions or you need help with something don't hesitate to contact me on slack, you will find me as _Zakariae El Mejdki_

File: index.html (hunk 7 above)
Comment (nikoescobal) [reply index.html]: Hi Zakariae! According to this article entitled "[HTML Style Guide and Coding Conventions,](https://www.w3schools.com/html/html5_syntax.asp)" it is recommended to always specify alt, width, and height for Images. As much as possible, we did not use inline styling, except for the case of images. We would appreciate your consideration on this matter.

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

File: index.html [hunk 11]
```diff
@@ -0,0 +1,670 @@
...
+              <div class="user-description">
+                <div class="page-name"></div>
+                <div class="align-center">
+                  <img
+                    class="images inline"
+                    src="assets/user-profile.jpg"
+                    alt="Page Profile"
+                    width="40"
+                    height="40"
+                  />
```
Comment (BrittanyBlake) [inline index.html]: please remove inline styling. The image tag should only contain the alt element as it is used for screen readers.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,670 @@
...
+            <div class="total-comments">
+              <div class="user-comments">
+                <div class="user-avatar">
+                  <img
+                    class="images"
+                    src="assets/user-avatar-1.jpg"
+                    alt="User Profile"
+                    width="40"
+                    height="40"
+                  />
```
Comment (BrittanyBlake) [inline index.html]: Please remove inline styling.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,670 @@
...
+              </div>
+              <div class="user-comments">
+                <div class="user-avatar">
+                  <img
+                    class="images"
+                    src="assets/user-avatar-2.jpg"
+                    alt="User Profile"
+                    width="40"
+                    height="40"
+                  />
```
Comment (BrittanyBlake) [inline index.html]: Please remove inline styling.

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @nikoescobal  and @mcihadkurel! 👋 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] The [HTML validator](https://validator.w3.org) is showing some errors on your page. Please fix these errors and resubmit.
Screen Shot 2020-06-11 at 6 14 38 PM

- [x] As was requested by the previous TSE, please remove inline styling from your HTML code and use your CSS file instead. I have highlighted some areas where you are using inline styling in my comments. Please use this is a guide and remove all other instances of inline styling. 

- [x] The icon of the user who uploaded the video and the users in the comments are not aligned properly. Please compare your clone with the original page and fix this.
Your clone 👇 
Screen Shot 2020-06-11 at 6 29 38 PM
The original page 👇 
Screen Shot 2020-06-11 at 6 29 42 PM
Your clone 👇 
Screen Shot 2020-06-11 at 6 29 57 PM
The original page 👇 
Screen Shot 2020-06-11 at 6 30 10 PM
****
### [OPTIONAL]

- [x] In your future projects, please add a more descriptive pull request that includes an appropriate title and a short summary of what you have done. 🗒️ 




****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by nikoescobal** [comment]
Hi @BrittanyBlake and @elmejdki! Thank you for taking time to review this project that @mcihadkurel and I spent time working on. We've applied all the suggested changes -- removed in-line styling, adjusted the CSS to match as closely as possible to match the original YouTube page, and have made sure that there are no errors on the validator.

File: style.css [hunk 14]
```diff
@@ -0,0 +1,287 @@
...
+  position: relative;
+  right: 35px;
+  top: 80px;
+}
+.image {
+  float: left;
+  height: 10px;
+  position: relative;
+  width: 168;
+  height: 94;
```
Comment (peterrobert) [inline style.css]: Remember width and height are units. This being the case you need to add a unit after the number. Example use px for pixels.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,469 @@
...
+              </div>
+              <div class="flex">
+                <i class="fa fa-thumbs-up icon-spacing" aria-hidden="true">
+                  115k</i
+                >
+
+                <i class="fa fa-thumbs-down icon-spacing" aria-hidden="true">
+                  5.8K</i
+                >
+                <i class="fa fa-share icon-spacing"> SHARE</i>
```
Comment (peterrobert) [inline index.html]: You need to make this a button. It is a requirement. You can wrap it using`<button>` tag

File: index.html [hunk 16]
```diff
@@ -0,0 +1,469 @@
...
+      </div>
+
+      <div id="right-navbar" class="flex">
+        <a href="#"><img src="assets/nav1.png" alt="" /></a>
+        <a href="#"><img src="assets/nav2.png" alt="" /></a>
+        <a href="#"><img src="assets/nav4.png" alt="" /></a>
+        <a href="#"><img src="assets/nav3.png" alt="" /></a>
+      </div>
+    </nav>
+
```
Comment (peterrobert) [inline index.html:38]: You need to wrap the main part of your webpage inside the main tag. You can check the usage of the semantic tag [here](https://www.w3schools.com/html/html5_semantic_elements.asp)

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle:  :stop_sign: 
Great work so far :+1:  Very good work implementing the changes required by the previous TSE. However, there are a few more issues that need fixing before I can approve this project :smiley: 

- [ ] Kindly add a margin between these two areas in your webpage :arrow_down: 
space

There are some more required changes below :arrow_down: 

Kindly work on the suggested areas and resubmit your work for another review

### [OPTIONAL ] :arrow_down: 
I can see you added stickler files and enabled the linter in your project. If you can kindly fix the linter errors. However, this is not a requirement for this project so feel free to leave it.

If you have any question concerning the review kindly contact me via slack on @peterrobert

### HAPPY CODING  :computer:

**Comment by nikoescobal** [comment]
Resolved latest request issues -- adjusted spacing between the video and sidebar, and added the main tag.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, you did a great job fixing all but one of the comments made by the previous TSE :+1: Before we can proceed, we need to fix the comments below:
- ```Sidebar``` has overlapped into the main video content. Let us fix it
margin
- Let us fix the stickler errors. Let view the details of the error and fix them

Kindly implement these changes and submit another core review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by christianotieno — CHANGES_REQUESTED** [review]
Hello Team, One crucial thing we have left out before we wrap this up. Kindly ensure that all the linter check are passing. While you have made the changes, please submit for another code review.

**Comment by nikoescobal** [comment]
Hi @christianotieno and @addod19 ! As @peterrobert mentioned, fixing the stickler errors is optional as it is not a project requirement. However, because I did enable it (it was my first time to install), I have decided to go ahead and fix all errors. @mcihadkurel and I are thankful to all the TSE's who took the time to review and help improve our code. Thank you.

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✔

## Impressive project team 👏🙌👌. Great job implementing the changes 🙌!

### It's a 👍 from. Go ahead and merge it.

### Onto the next project 🚀. Best of luck ✨.

Happy coding 👨‍💻

**Pull request merged**